tests/unit/test_aiopg_connector.py

```diff
@@ -52,6 +52,55 @@ async def corofunc(a, b):
     assert await corofunc(1, 2) == (1, 2)
 
 
+@pytest.mark.asyncio
+async def test_wrap_query_exceptions_reached_max_tries(mocker):
+    @aiopg_connector.wrap_query_exceptions
+    async def corofunc(connector):
+        raise psycopg2.errors.OperationalError(
+            "server closed the connection unexpectedly"
+        )
```

Comment:
Can we count that we were called 6 times ?

Reply:
Done.